web/modules/custom/joinup_invite/src/InvitationMessageHelper.php

```diff
@@ -93,8 +93,14 @@ protected function getDefaultArguments(InvitationInterface $invitation): array {
       $arguments["@invitation:${action}_url"] = Url::fromRoute('joinup_invite.update_invitation', $url_arguments, $url_options)->toString();
     }
 
-    $arguments['@invitation:role'] = $invitation->field_invitation_og_role->isEmpty() ? '' :
-      strtolower($invitation->field_invitation_og_role->first()->entity->label());
+    $arguments['@invitation:role'] = '';
+    if ($invitation->get('field_invitation_og_role')->isEmpty()) {
+      return $arguments;
+    }
+    // Field is not empty so at least one value exists.
+    elseif ($entity = $invitation->field_invitation_og_role->first()->entity) {
+      $arguments['@invitation:role'] = $entity->label();
+    }
```

Comment:
Same here, if we pass in an invitation bundle that doesn't have this field (like e.g. the `discussion` bundle) then this will throw a fatal error.

We should also not chain the `->entity` call, this can cause a notice `accessing member property on NULL` to be thrown in case the field exists but returns a `NULL` value. This can happen e.g. when the collection is deleted in the same request.